packages/scaffolding-cli/src/domain/config/questions.ts

```diff
@@ -58,7 +58,7 @@ export function cliQuestions(default_project_name: string): Array<PromptQuestion
         {
             "type": "select",
             "name": "platform",
-            "message": "Select Target Platform",
+            "message": "Define target platform",
```

Comment:
If it's a list, does define make sense?

Reply:
I struggled with this a bit. Because you ARE selecting, but not from a static list... you have to type what you want to use (meaning there's room for error... like a freetext box). On balance though, select is probably the closest.